.github/workflows/test.yml

```diff
@@ -107,7 +107,7 @@ jobs:
                     curl -sL "https://github.com/goss-org/goss/releases/download/${{ env.GOSSVERSION }}/goss-${GOSS_ARCH}" -o goss
                     chmod +x goss
                     go build cmd/tests/test-features.go
-                    ./test-features -f ${{ matrix.flavour }} -i test-image-${{ matrix.flavour }}-${{ matrix.arch}}:local -p ${{ matrix.arch }} -t ${PWD}/.testbed -g ${PWD}/goss -l debug
+                    ./test-features -c -f ${{ matrix.flavour }} -i test-image-${{ matrix.flavour }}-${{ matrix.arch}}:local -p ${{ matrix.arch }} -t ${PWD}/.testbed -g ${PWD}/goss -l debug
```

Comment:
SCR: Please pick your style consistent:  I guess ${{ matrix.arch}} misses a blank.
I'd also like every variable wrote in this style ${{matrix.arch}} for the looks here (no blanks breaking the flow)
but that's your choice.

Reply:
Thanks. Added the whitespace. I'd like to use the whitespace as the official Github Actions docs also use whitespaces and it also distinguishes the github actions variable expansions additionally from environment variables (e.g. `${{ matrix.arch }}` vs. `${PWD}` in this example)